modules/private-clusters-setting-dns-private.adoc

```diff
@@ -4,13 +4,22 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="private-clusters-setting-dns-private_{context}"]
-= Setting DNS to private
+= Configuring DNS records to be published in a private zone
 
-After you deploy a cluster, you can modify its DNS to use only a private zone.
+For all {product-title} clusters, whether public or private, DNS records are published in a public zone by default.
+
```

Comment:
In my opinion, we want to start with the case where they are configuring DNS to be private, and follow it by the case where they don't want it to be private after all.